src/components/MarkDown/MarkdownEditor/MarkDownEditor.constants.ts

```diff
@@ -0,0 +1,14 @@
+export enum MARKDOWN_EDITOR_TYPES {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary'
+}
+
+export enum MARKDOWN_EDITOR_MODES {
+	EDIT = 'edit',
```

Comment:
looks like formatting is slightly off here. On my IDE (vscode), there is a format on save setting which will ensure the correct spacings are applied. Might be good to use something like this to avoid these formatting issues?

<img width="901" alt="Screenshot 2022-09-30 at 15 00 36" src="https://user-images.githubusercontent.com/39112648/193286549-d70e890a-7238-4305-a9cc-8f04dc918a42.png">